plugin/src/main/java/at/helpch/chatchat/task/DataSaveTask.java

```diff
@@ -0,0 +1,26 @@
+package at.helpch.chatchat.task;
+
+import at.helpch.chatchat.ChatChatPlugin;
+import at.helpch.chatchat.api.ChatUser;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+
+public class DataSaveTask extends BukkitRunnable {
```

Comment:
this class can be removed in favor of the lambda argument in runTaskTimer. this can all be converted into a single lamda expression using streams too